dali/kernels/slice/slice_cpu.h

```diff
@@ -355,11 +346,8 @@ void SliceKernel(SequentialExecutionEngine &exec_engine,
                  const TensorShape<Dims> &out_shape,
                  const OutputType *fill_values,
                  int channel_dim = -1,  // negative if no channel dim or already processed
-                 int min_blk_sz = 16000,
-                 int req_nblocks = -1) {
+                 int = -1, int = -1) {
```

Comment:
Nitpick: I'd keep the names in comments.
```suggestion
                 int /* min_blk_sz */ = -1, int /* req_nblocks */ = -1) {
```